code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -356,16 +356,65 @@
 	new /obj/item/crowbar(src)
 
 /*
- * Duffelbags - My thanks to MrSnapWalk for the original icon and Neinhaus for the job variants - Dave.
+ * Duffelbags
  */
 
 /obj/item/storage/backpack/duffel
 	name = "duffelbag"
-	desc = "A large grey duffelbag designed to hold more items than a regular bag."
+	desc = "A large grey duffelbag designed to hold more items than a regular bag. It slows you down when unzipped."
 	icon_state = "duffel"
 	item_state = "duffel"
 	max_combined_w_class = 30
-	slowdown = 1
+	/// Is the bag zipped up?
+	var/zipped = TRUE
+	/// How long it takes to toggle the zip state of this bag
+	var/zip_time = 0.7 SECONDS
+
+/obj/item/storage/backpack/duffel/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It is currently [zipped ? "zipped" : "unzipped"]. Ctrl+click to [zipped ? "un-" : ""]zip it!</span>"
+
+/obj/item/storage/backpack/duffel/CtrlClick(mob/user)
+	. = ..()
+	handle_zipping(user)
```

Comment:
The only problem I have is that trying to drag the duffel while it's on the floor, always attempts to zip/unzip the bag. Have you considered another hotkey like AltShift, or even using `attack_self`?